test/Microsoft.Health.Fhir.Tests.E2E/Rest/Sql/SchemaTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Microsoft.Health.Fhir.Web;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.Tests.E2E.Rest.Sql
+{
+    public class SchemaTests : IClassFixture<HttpIntegrationTestFixture<Startup>>
+    {
+        private readonly HttpClient _client;
+
+        public SchemaTests(HttpIntegrationTestFixture<Startup> fixture)
+        {
+            _client = fixture.HttpClient;
+        }
+
+        public static IEnumerable<object[]> Data =>
+            new List<object[]>
+            {
+                new object[] { "_schema/compatibility" },
+                new object[] { "_schema/versions" },
+                new object[] { "_schema/versions/current" },
+                new object[] { "_schema/versions/123/script" },
+            };
+
+        [Theory]
+        [MemberData(nameof(Data))]
+        public async Task WhenRequestingSchema_GivenGetMethod_TheServerShouldReturnNotImplemented(string path)
+        {
+            await SendAndVerifyStatusCode(HttpMethod.Get, path, HttpStatusCode.NotImplemented);
+        }
+
+        // Investigate why these return 415 in the test project, but 404 when running in postman
```

Comment:
Investigate this.

Reply:
I'm going to punt this until we're actually running these tests against a deployed instance instead of only `TestServer`.